doc/concepts/configuration.rst

```diff
@@ -0,0 +1,676 @@
+
+..  include:: /concepts/configuration/configuration_etcd.rst
+    :start-after: ee_note_etcd_start
+    :end-before: ee_note_etcd_end
+
+Tarantool enables you to store configuration data in one reliable place using `etcd <https://etcd.io/>`_.
+To achieve this, you need to:
+
+1.   Provide a local YAML configuration with an etcd endpoint address and key prefix in the ``config`` section:
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/etcd/config.yaml
+        :language: yaml
+        :dedent:
+
+2.   Publish a cluster's configuration to an etcd server.
+
+Learn more from the following guide: :ref:`Storing configuration in etcd <configuration_etcd>`.
+
+
+..  _configuration_precedence:
+
+Configuration precedence
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tarantool configuration options are applied from multiple sources with the following precedence, from highest to lowest:
+
+-   `TT_*` :ref:`environment variables <configuration_environment_variable>`.
+-   Configuration from a :ref:`local YAML file <configuration_file>`.
+-   :ref:`Centralized configuration <configuration_etcd_overview>` stored in etcd.
+-   `TT_*_DEFAULT` :ref:`environment variables <configuration_environment_variable>`.
+
+If the same option is defined in two or more locations, the option with the highest precedence is applied.
+
+
+
+..  _configuration_options_overview:
+
+Configuration options overview
+------------------------------
+
+This section gives an overview of some useful configuration options.
+All the available options are documented in the :ref:`Configuration reference <configuration_reference>`.
+
+.. _configuration_options_connection:
+
+Connection settings
+~~~~~~~~~~~~~~~~~~~
+
+To configure an address used to listen for incoming requests, use the ``iproto.listen`` option.
+Below are a few examples on how to do this:
+
+*   Set a listening port to ``3301``:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "3301"
+
+*   Set a listening address to ``127.0.0.1:3301``:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301"
+
+
+*   Configure several listening addresses:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301,127.0.0.1:3303"
+
+*   Enables :ref:`traffic encryption <enterprise-iproto-encryption>` for a connection using corresponding URI parameters:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301?transport=ssl&ssl_key_file=localhost.key&ssl_cert_file=localhost.crt&ssl_ca_file=ca.crt"
+
+    Note that traffic encryption is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+*   Use a Unix domain socket:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "unix/:./var/run/{{ instance_name }}/tarantool.iproto"
+
+
+.. _configuration_options_access_control:
+
+Access control
+~~~~~~~~~~~~~~
+
+The ``credentials`` section allows you to grant the specified privileges to users.
```

Comment:
It may be helpful to say here that this config _creates_ the users. It's not obvious. I thought that perhaps I should create them somehow and then run this config to grant roles.